transport/node.go

```diff
@@ -0,0 +1,29 @@
+package transport
+
+import (
+	"go.uber.org/atomic"
+)
+
+type nodeStatus = atomic.Bool
+
+const (
+	statusDown = false
+	statusUP   = true
+)
+
+type Node struct {
+	addr       string
+	datacenter string
+	rack       string
+	tokens     []byte // It should be []Token, but we don't have parsing for that yet.
+	pool       *ConnPool
+	status     *nodeStatus
```

Comment:
Why pointer here?

Reply:
"go.uber.org/atomic" package implements methods on pointers, like:
    func NewBool(val bool) *atomic.Bool
    func (x *Bool) Load() bool

So we need pointers. The question is, is it better to have:
    type nodeStatus = atomic.Bool
    type Node struct {
        ...
        status *nodeStatus
    }
or
    type nodeStatus = *atomic.Bool
    type Node struct {
        ...
        status nodeStatus
    }